packages/design-system/src/components/Badge/variants/BadgePopover.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { forwardRef, Ref } from 'react';
+
+import Clickable from '../../Clickable';
+import Divider from '../../Divider';
+import { StackHorizontal } from '../../Stack';
+import BadgePrimitive, { BadgePopoverItem, BadgePrimitiveProps } from '../primitive/BadgePrimitive';
+
+import classnames from 'classnames';
+import styles from './BadgePopover.module.scss';
+
+// --------------------------------------------------
+// Badge Popover button
+// --------------------------------------------------
+
+interface BadgePopoverButtonProps {
+	componentId?: string;
```

Comment:
Why `component` prefix on `id` ?

Reply:
Because "id" is keyword in HTML so i don't want to introduce misunderstanding.